apps/discord-bot/src/commands/minecraft/available.command.ts

```diff
@@ -0,0 +1,44 @@
+import { MojangPlayerArgument } from '#arguments';
+import { SUCCESS_COLOR, WARNING_COLOR } from '#constants';
+import { MojangApiService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+
+@Command({ description: 'commands.available', args: [new MojangPlayerArgument()] })
+export class AvailableCommand {
+  public constructor(private readonly mojangApiService: MojangApiService) {}
+
+  public async run(context: CommandContext) {
+    const name = context.option<string>('player');
```

Comment:
Add a check for if the name is actually a valid minecraft name. Currently we do something like `!name.match(/^\w+$/i) || name.length > 16`